### PR TITLE
stop on all already existing files (disk&hk)

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds a fix for the leaking coffee machine.

**How to prepare for test**:
- [x] install on stage
- [x] activate stage: `us`

**How to test file already on disk**:
- [x] find a case that already been uploaded to scout after 2019-11-27
- [x] run following command: `cg upload scout CASE-ID`

**Expected test outcome**:
- [x] the command should exit with a warning

**How to test file already in hk**:
- [x] find a case that already been uploaded to scout after 2019-11-27
- [x] remove the case scout load config on disk (not in hk)
- [x] run following command: `cg upload scout CASE-ID`

**Expected test outcome**:
- [x] the command should exit with a warning

- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt  
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because we change the logic a bit but in a transparent way from the automation side

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
image
👍

File: cg/cli/upload.py [hunk 1]
```diff
@@ -433,15 +433,19 @@ def scout(context, re_upload, print_console, case_id):
...
+        context.abort()
 
     scout_upload_api.save_config_file(scout_config, file_path)
     hk_api = context.obj['housekeeper_api']
     try:
         LOG.info("Upload file to housekeeper: %s", file_path)
         scout_upload_api.add_scout_config_to_hk(file_path, hk_api, case_id)
     except FileExistsError as err:
-        LOG.warning(err)
+        LOG.warning(str(err) + ", consider removing the file from housekeeper and try again")
```
Comment (ingkebil) [inline cg/cli/upload.py]:
```suggestion
        LOG.warning("%s, consider removing the file from housekeeper and try again", str(err))
```

Comment (patrikgrenfeldt) [reply cg/cli/upload.py]: good catch!

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**